gtsfm/common/image.py

```diff
@@ -19,6 +19,7 @@ class Image(NamedTuple):
     sensor_width_db = SensorWidthDatabase()
     file_name: Optional[str] = None
     mask: Optional[np.ndarray] = None
+    default_focal_length_factor: float = 1.2
```

Comment:
Thanks for the PR, Yanwei!

Could we make this a global variable, e.g. `DEFAULT_FOCAL_LENGTH_FACTOR = 1.2`, and define its units, and why we chose a value of 1.2 in an inline comment?

Reply:
Done